From 3952c9c66cc27f40e7342db9c29fa56c4b00e4bc Mon Sep 17 00:00:00 2001
From: Tudor Ambarus <tudor.ambarus@microchip.com>
Date: Mon, 7 Oct 2019 11:53:46 +0300
Subject: [PATCH 081/471] crypto: atmel: Add support for the AES IP available
 on sam9x60

Add version for the AES IP available on the sam9x60 SoC.

Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
[ta: ported from 4.19 to 5.4]
Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
[ta: cherry picked to v5.10]
Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
[ta: cherry picked to v5.15, drop fall through comment]
Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
---
 drivers/crypto/atmel-aes.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/crypto/atmel-aes.c b/drivers/crypto/atmel-aes.c
index f72c6b3e4ad8..f113d565918d 100644
--- a/drivers/crypto/atmel-aes.c
+++ b/drivers/crypto/atmel-aes.c
@@ -2510,6 +2510,7 @@ static void atmel_aes_get_cap(struct atmel_aes_dev *dd)
 	/* keep only major version number */
 	switch (dd->hw_version & 0xff0) {
 	case 0x700:
+	case 0x600:
 	case 0x500:
 		dd->caps.has_dualbuff = 1;
 		dd->caps.has_cfb64 = 1;
-- 
2.34.1

